ingame_manuals/malf_ai.html

```diff
@@ -1,23 +1,100 @@
-
-<h2>Software</h2><br>
-Software are abilities that have to be unlocked via research menu (Hardware tab). Unlocked abilities appear in Software tab. <b>Sometimes, abilities won't appear in this tab after being researched. This can be fixed by relogging or using "ai-core" command.</b> Abilities are tiered, T1 being the weakest ones, while T4 are strongest ones. To reach higher tier you have to research abilities in lower tier. We currently have 12 abilities, in 3 research trees:<br>
-<b>Networking</b> - Hacking-oriented abilities. T1 ability is Basic Encryption Hack, which allows you to hack more APCs. Higher tiers allow faking centcom messages, and even setting alert level. T4 ability is System Override, that rapidly hacks remaining APCs and gives you access to station self destruct sequence.<br>
-<b>Interdiction</b> - Sabotage-oriented abilities. T1 ability allows you to recall emergency shuttle. T2 allow you to unlock cyborgs even without access to robotics console. T3 allows you to hack a random maintenance drone and slave it to your will. T4 allows you to hack other cyborgs and salve them under your control. and T4 ability allows you to hack other AIs to slave them under your control.<br>
-<b>Manipulation</b> - Physical-oriented abilities. T1 ability allows you to break few lights, and rarely even APCs. T2 ability allows you to apply upgrade of your choice to camera, or reactivate broken camera. T3 ability allows you to create weak forcefield, that holds air, but won't last for long.T4 ability allows you to create localized gravity malfunction that reverses and throws people in the area really hard. And T5 ability allows you to overload machines, detonating them in weak explosion.<br>
-<b>Synthetic Takeover</b> - Abilities revolving around you and your minions. T1 ability allows you to reset your cyborgs module. T2 ability allows you to hack IPCs. T3 ability allows you to make your cyborgs stronger And T4 ability allows you to take over the stations electronics and unlocks a human to cyborg converting machine.<br>
-<h2>End</h2><br>
-If you still have some questions, either check the wiki, ask on IRC, or adminhelp and ask your friendly administration staff.
\ No newline at end of file
+<html>
+<head>
+    <title>Malfunction Guide</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+</head>
+<body>
+    <div class="main">
+    <p>
+        <a style="cursor: pointer; color: #00a0fc;" onclick=" document.querySelector('body').classList.contains('dark') ? document.querySelector('body').classList.remove('dark') : document.querySelector('body').classList.add('dark')" title="Switch to light or dark theme">🌓 Light|dark mode</a>
+    </p>
+    <h1>Malfunctioning AI guide</h1><br>
+
+    <p style="margin-left:2em">This guide contains most important OOC information for malfunctioning AIs.</p>
```

Comment:
Why there is snowflake margin left styles? If you have style tag, then put it inside it.